fe/fe-core/src/main/java/com/starrocks/common/util/concurrent/lock/Locker.java

```diff
@@ -213,9 +213,11 @@ public void lockDatabases(List<Database> dbs, LockType lockType) {
         if (dbs == null) {
             return;
         }
-        dbs.sort(Comparator.comparingLong(Database::getId));
-        for (Database db : dbs) {
-            lockDatabase(db, lockType);
+        // use TreeMap to remove duplicate and sort by db id
```

Comment:
Maybe we can use Set as input?

Reply:
because the equals of Database is not efficient, so here use dbid to do distinct and sort.